xsd/netex_framework/netex_reusableComponents/netex_environment_support.xsd

```diff
@@ -449,6 +449,16 @@ Rail transport, Roads and Road transport
 			<xsd:enumeration value="covered"/>
 			<xsd:enumeration value="mixed"/>
 			<xsd:enumeration value="unknown"/>
+			<xsd:enumeration value="true">
+				<xsd:annotation>
+					<xsd:documentation>Value to be used, if covered is used as a "boolean". Equals covered. +v2.0</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="false">
+				<xsd:annotation>
+					<xsd:documentation>Value to be used, if covered is used as a "boolean". Equals outdoors. +v2.0</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
```

Comment:
Should there be noted that this usage is deprecated?

Reply:
done